### PR TITLE
Ttab 25

### PR DESCRIPTION
**Pull request by AshleyThomas87**
footer updates
footer

### CODE REVIEW

File: src/Components/Footer/style.jsx [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
+  position: fixed;
   bottom: 0;
   left: 0;
   width: 100%;
   display: flex;
   justify-content: space-between;
   align-items: center;
-  background: burlywood;
-  padding: 25px 15px;
+  background: #030305;
```
Comment (jmclain-banyan) [inline src/Components/Footer/style.jsx:11]: lets grab this color from our colors object in `src/styles/colors.js`

File: src/Components/Footer/style.jsx [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
+  padding: 0.2rem calc((100vw - 1000px) / 2);
   max-height: 30px;
 `;
 
 export const AppTitle = styled.div`
-  font-size: 22px;
+  font-family: 'Roboto';
   font-weight: 400;
+  color: #fefefe;
+  font-size: clamp(18px, 4vw, 18px);
```
Comment (jmclain-banyan) [inline src/Components/Footer/style.jsx:20]: for the font-size lets set the max to 22px

File: src/Components/Footer/style.jsx [hunk 3]
```diff
@@ -1,25 +1,29 @@
...
   cursor: pointer;
 `;
 export const CompanyWrapper = styled.div`
-  font-size: 18px;
+  font-family: 'Roboto';
   font-weight: 500;
+  color: #fefefe;
+  font-size: clamp(18px, 4vw, 18px);
```
Comment (jmclain-banyan) [inline src/Components/Footer/style.jsx:27]: for the font-size lets set the min to 14px and max 22px

**Review by jmclain-banyan — CHANGES_REQUESTED** [review]
overall looks good, just a couple small changes

**Comment by AshleyThomas87** [comment]
made new updates

**Review by jmclain-banyan — APPROVED** [review]

**Pull request merged**